src/plugins/petpet/index.ts

```diff
@@ -177,6 +178,8 @@ export default definePlugin({
                 }
 
                 gif.finish();
+                // FIXME: vvvvvvv
+                // @ts-ignore errors on some ts versions, works fine
```

Comment:
always prefer expect-error over ignore and what does this even mean

```suggestion
                // @ts-expect-error errors on some ts versions, works fine
```

Reply:
it errors on some typescript versions, but works fine on others

Reply:
updated the comment to be better